app/src/main/java/com/example/a3048_carapp/MainActivity.kt

```diff
@@ -4,7 +4,7 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 
 class MainActivity : AppCompatActivity() {
-    override fun onCreate(savedInstanceState: Bundle?) {
+    override fun onCreate(savedInstanceState:Bundle?) {
```

Comment:
Space after colon removed to adhere to the Kotlin horizontal whitespace conventions. 
https://kotlinlang.org/docs/coding-conventions.html#colon